### PR TITLE
Add geth version requirements to clef documentation

### LINKED ISSUE #2221: Add minimum version of clef to the docs
See https://discordapp.com/channels/411401661714792449/748185700515446814/750695944781037670.

Some endpoints used weren't available until v1.9

### PR DESCRIPTION
**Pull request by derekpierre**
Fixes #2221

### CODE REVIEW

**Review by KPrasch — COMMENTED** [review]
What do you think about including a link to the clef docs? 

https://geth.ethereum.org/docs/clef/tutorial

**Comment by derekpierre** [comment]
It feels like we were duplicating links for clef. How about we use your link instead of the github link in the `Signing with Clef` section...?

**Comment by KPrasch** [comment]
@derekpierre - Yeah - that's even better. :+1:

**Review by KPrasch — APPROVED** [review]

**Comment by KPrasch** [comment]
:cowboy_hat_face:

File: docs/source/guides/ethereum_node.rst [hunk 1]
```diff
@@ -130,19 +130,22 @@ Signing with Clef
 
 Clef enables applications to connect to an ethereum node and send locally signed
 transactions to be broadcasted. More
-information about Clef can be found `here <https://github.com/ethereum/go-ethereum/tree/master/cmd/clef>`_. Clef can
+information about Clef can be found `here <https://geth.ethereum.org/docs/getting-started>`_. Clef can
```
Comment (cygnusv) [inline docs/source/guides/ethereum_node.rst:133]:
```suggestion
information about Clef can be found `here <https://geth.ethereum.org/docs/clef/tutorial>`_. Clef can
```

Comment (derekpierre) [reply docs/source/guides/ethereum_node.rst]: Thanks too much copy and pasting 😅

**Pull request merged**